projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.html

```diff
@@ -1,8 +1,8 @@
 <div class="resource-view" *ngIf="resource">
 
   <!-- dsp-property-view -->
-  <div *ngIf="propArray.length !== 0; else noProperty">
-    <dsp-property-view [parentResource]="resource" [propArray]="propArray" [systemPropArray]="systemPropArray">
+  <div *ngIf="resPropInfoVals.length !== 0 || systemPropDefs.length !== 0; else noProperty">
```

Comment:
Is this check necessary? How would `PropertyViewComponent` behave if both `resPropInfoVals` and `systemPropDefs` are empty arrays?

You don't have to worry about not initialised members because bot arrays are initialised with an empty array. I think that's a good practice.

Reply:
ok so I will remove this check

Reply:
Ok, fine with me if you leave this check.

Reply:
I mean, the current way it is implemented is fine :-)